test/helper.ts

```diff
@@ -4,13 +4,17 @@ import { bsv } from '../src/utils';
 import { ContractDescription } from '../src/contract';
 import { compile, CompileResult, getPlatformScryptc} from '../src/compilerWrapper';
 export function loadDescription(fileName: string): ContractDescription {
-  return JSON.parse(readFileSync(join(__dirname, 'fixture', fileName.replace('.scrypt', '_desc.json'))).toString());
+  return JSON.parse(readFileSync(join(__dirname, "../out/", fileName)).toString());
 }
 
 export function loadASM(fileName: string): string {
   return readFileSync(join(__dirname, 'fixture', fileName.replace('.scrypt', '_asm.json'))).toString();
 }
 
+export function loadFile(fileName: string): string {
```

Comment:
rename to getContractFilePath

Reply:
bbb5e69d6e4755c8e0ad447e245a133a66923992